src/connectionbenchmark/connectionbenchmark.cpp

```diff
@@ -95,8 +97,6 @@ void ConnectionBenchmark::start() {
           [this, downloadTask]() { m_benchmarkTasks.removeOne(downloadTask); });
   m_benchmarkTasks.append(downloadTask);
   TaskScheduler::scheduleTask(downloadTask);
-
-  setState(StateRunning);
```

Comment:
why this change?

Reply:
So that we have set `m_state == StateRunning` in case `handleStabilitiChange()` gets called in `L79`.